retrofit-converters/moshi/src/main/java/retrofit/MoshiConverter.java

```diff
@@ -38,7 +38,7 @@ public MoshiConverter() {
   public MoshiConverter(Moshi moshi) {
     if (moshi == null) throw new NullPointerException("moshi == null");
     this.moshi = moshi;
-    this.mediaType = MediaType.parse("application/json; charset=utf-8");
+    this.mediaType = MediaType.parse("application/json; charset=UTF-8");
```

Comment:
(Probably worthwhile to grep and make sure 100% of your projects are consistent.)